### PR TITLE
enh(UI): Clear filter by one click on close icon into search/filter bar

### PR DESCRIPTION
**Pull request by Jordan91**
## Description

For now you can't clear easily the search filter, the feature expected is to add Close Icon to allow the user clear filter by one click

**Fixes** #

Add Close Icon into search/filter bar , to allow the user to clear easily the filter

https://www.loom.com/share/d984613c61ce471cb69e8ebe19c9ede1

## Type of change

- [x] New functionality (non-breaking change)

## Target serie

- [x] 21.10.x (master)

<h2> How this pull request can be tested ? </h2>

Go to the ressource status and start to set a filter by selecting criterias. After you can try to delete your setting by clicking on the Close icon at the end of this search bar.

### CODE REVIEW

File: lang/fr_FR.UTF-8/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -16240,5 +16240,6 @@ msgstr "Détails du Contact"
 msgid "Service graphs"
 msgstr "Graphiques de services"
 
-msgid "Enable the one-click export button for poller configuration [BETA]"
-msgstr "Activer le bouton d’export de la configuration des collecteurs en un seul clic [BETA]"
+msgid "Clear filter"
+msgstr "Réinitialiser le filtre"
```
Comment (Thebarda) [inline lang/fr_FR.UTF-8/LC_MESSAGES/messages.po:16244]:
```suggestion
msgstr "Effacer le filtre"
```

File: www/front_src/src/Resources/Filter/index.tsx [hunk 2]
```diff
@@ -48,6 +50,10 @@ const useStyles = makeStyles((theme) => ({
   autocompletePopper: {
     zIndex: theme.zIndex.tooltip,
   },
+  clearFilter: {
+    padding: 'revert',
+    position: 'inherit',
```
Comment (Thebarda) [inline www/front_src/src/Resources/Filter/index.tsx:55]: Do you need this ?

Comment (Jordan91) [reply www/front_src/src/Resources/Filter/index.tsx:55]: i think not

File: www/front_src/src/Resources/Filter/index.tsx [hunk 3]
```diff
@@ -48,6 +50,7 @@ const useStyles = makeStyles((theme) => ({
   autocompletePopper: {
     zIndex: theme.zIndex.tooltip,
   },
+
```
Comment (Thebarda) [inline www/front_src/src/Resources/Filter/index.tsx:53]:
```suggestion
```

**Review by jdelpierre — APPROVED** [review]

**Review by Thebarda — APPROVED** [review]

**Pull request merged**